srcjs/widget.js

```diff
@@ -90,9 +91,10 @@ export function reactWidget(name, type, components, options) {
                         }
                         lastValue = value;
                     }
-                    ReactDOM.render(hydrate(components, value.tag), el);
+                    this.instance.component = ReactDOM.render(hydrate(components, value.tag), el);
```

Comment:
@alandipert I still think we want to do this, but I learned that this will be `null` with functional (stateless) components.  I guess I can add a comment here with some warning.

Reply:
added comment